imdbsol.txt

```diff
@@ -1,7 +1,46 @@
 #!/bin/bash
 
-t=`cat imdb.txt`
-rating="${t#*imdbRating}"
-rating="${rating%%%%}"
-res=${rating:2:3}	
-echo $res
+mkdir tmp1
```

Comment:
You don't need to create this directory. All *Nix systems have a directory at `/tmp` which can be used for this purpose. http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/tmp.html

But having said that, most of the things that you are doing in your script - you dont need files for that. you can do that using variables and pipes.